wire.py

```diff
@@ -0,0 +1,42 @@
+from PySide2.QtCore import *
+from PySide2.QtGui import *
+from PySide2.QtWidgets import *
+from PySide2.QtGui import *
+
+from gui_views.dialogs import *
+
+class Wire(QGraphicsLineItem):
+    def __init__(self, line, pen, state):
+        super(Wire, self).__init__()
+        # set pen and line passed in
+        self.setPen(pen)
+        self.setLine(line)
+        self.parent_key = None
+        self.child_key = None
+        self.state = state
+
+    def mousePressEvent(self, event):
```

Comment:
i'm wondering if this should be double click instead of single click because if the user is trying to drag the object they might unintentionally get the delete connection dialog

Reply:
changed it to double click